mipengine/controller/algorithm_executor_nodes.py

```diff
@@ -20,6 +21,11 @@
 from mipengine.node_tasks_DTOs import UDFPosArguments
 
 
+class AsyncResult:
+    def get(self, timeout=None):
+        pass
```

Comment:
Yes, it is for type annotation only, but we are not importing `celery.result.AsyncResult` because it is not needed. I m also not sure what is the correct way to do this.

Reply:
What's the point of type annotating with a class that's not actually returned?

Reply:
This is what I mean "I m not sure what is the correct way to do this". I want to annotate(essentially show to future me or a fellow dev) that method `queue_run_udf` returns an object named AsyncResult that has a method named `get`, but without importing such a class, since, apart from type annotation, importing such a class is not needed. That's the thing with duck(dynamic) typing languages, that you cannot easily communicate in the code the types of the objects being used. If you have a better knowledge on how to do it, bring it on.